### ISSUE TITLE
Parellell loading of asset & instances from plugins

### ISSUE THREAD

**Issue by AnnaSasDev** [AterraEngine/aterra_engine-cs#11]
### General Info
For speed optimization I want to parallellize the workload of asset loading.
This will have some effect on "loading screens" as there isn't a built in system to currently track the completed workload... will have to work on this before we can finish writing this user story

---
#### Functional Requirements
- [ ] ?

#### Technical Requirements
- [ ] ?

---
### Tasks
- [ ] ?

**Comment by AnnaSasDev** [comment]
Duplicate of #42
